### DISCUSSION TITLE
Security

### DISCUSSION THREAD

**Discussion by ApexMuse** [DarkaOnLine/L5-Swagger#353] [Q&A]
I'm a bit of a noob, so please forgive me if this question is rather ignorant/uninformed.

I'm using Sanctum and have a security scheme configured so that a user can enter an auth bearer token in order to use the API routes in the docs.  Is there a way to **_automatically_** add a token so that it doesn't need to be entered every time you load the docs? Here's what I have so far:

`'sanctum' => [ // Unique name of security
    'type' => 'apiKey', 
    'description' => '<strong>Enter token in format (Bearer {token})</strong>',
    'name' => 'Authorization', 
    'in' => 'header', 
],`

` security={{"sanctum": {}}},`

**Comment by DarkaOnLine** [comment]
Have you tried looking into #318 ?
